### DISCUSSION TITLE
Make bar monochrome

### DISCUSSION THREAD

**Discussion by wytsai7660** [fastfetch-cli/fastfetch#960] [Q&A] [answered]
Hello. Is there any way to recreate the effect like this in these newer versions?

![upload_ec389308c9b5f0fd1b1bb167ec8f73a0](https://github.com/fastfetch-cli/fastfetch/assets/116703995/826e0d3e-6f6d-4017-8510-b47b91a0cdef)

This is the current appearance:

<img width="922" alt="Screenshot 2024-05-24 at 01 30 01" src="https://github.com/fastfetch-cli/fastfetch/assets/116703995/c2e021b0-e99d-4b59-8d4c-9aa7a5e56865">

It always displays those three colors (red, yellow, green) instead of displaying only the maximum one.

**Comment by CarterLi** [comment]
Not yet. The closest option is to set those three colors to the same color.

```jsonc
  "display": {
    "percent": {
        "color": {
            "green": "color",
            "yellow": "color",
            "red": "color"
        }
    }
  }
```

**Comment by CarterLi** [comment, marked as answer]
Supported in the latest dev branch. Please test it with `fastfetch --percent-type 10 # 10 = 2 (bar) + 8 (monochrome)`

**Reply by wytsai7660** [reply]
Yes! This works fine on my machine. Thank you very much for the instant reply and fix! 🙏

<img width="922" alt="Screenshot 2024-05-24 at 11 33 35" src="https://github.com/fastfetch-cli/fastfetch/assets/116703995/dcd1108c-d2c6-469c-a6dd-0cd279cbe1d2">
